behat.yml.dist

```diff
@@ -11,7 +11,7 @@ default:
       filters:
         tags: "@blackbox"
   extensions:
-    Behat\MinkExtension:
+    Drupal\MinkExtension:
```

Comment:
Could this change use more documentation in the release notes?

Reply:
Good call! I've update https://github.com/jhedstrom/drupalextension/releases/tag/v4.0.0alpha3